lib/strip_attributes.rb

```diff
@@ -5,7 +5,12 @@ module ActiveModel::Validations::HelperMethods
   def strip_attributes(options = nil)
     StripAttributes.validate_options(options)
 
-    before_validation do |record|
+    rails_opts = {
+      if: options && options[:if],
+      unless: options && options[:unless]
+    }
+
+    before_validation(rails_opts) do |record|
```

Comment:
[KUDOS] I love, love, love 😍this as an implementation decision. It make me way more comfortable about this feature and this PR. Great idea to just pass the options to `before_validation`. I hadn't thought of that, but it makes total sense.

[PLEASE CHANGE] However, I have a code cleanup request before merging. Please get rid of the `rails_opts` variable and change this to:

```ruby
before_validation(options.slice(:if, :unless)) do |record|
  StripAttributes.strip(record, options)
end
```